PCU/src/main/java/com/coffeelovers/pcu/PcuApplication.java

```diff
@@ -32,7 +32,7 @@ ApplicationRunner init(
 			ActorRepository actorRepository, 
 			VariableCostsRepository variableCostsRepository, 
 			InvoiceRepository invoiceRepository,
-			RawMaterialRepository rawMaterialRepository
+			InvoiceDetailsRepository rawMaterialRepository
```

Comment:
Are we not changing the rawmaterialRepository variable name to match the class?
```suggestion
			InvoiceDetailsRepository invoiceDetailsRepository
```

Reply:
wait my friend, i have done all these stuff(tables, Repositories, classes.. ). no need to update any part of  backend  from your side!!

Reply:
No all classes and repositories are working constantly with database, so no need to work on them. Please, keep just working on front-end.
I will push what I have done after the today's class. Now I am testing all the back-end components